<?xml version="1.0"?>
<launch>
  <arg name="pure_pursuit_config" default="$(find neato_planner)/config/pure_pursuit.yaml" />
  <rosparam file="$(arg pure_pursuit_config)" command="load"/>
  <node pkg="neato_planner" name="pure_pursuit_node" type="pure_pursuit.py"  output="screen">
      <param name="waypoints_filepath" value="$(find neato_planner)/waypoints/patrol.csv"/> 
  </node>
  <node pkg="neato_planner" name="visualize_markers_node" type="visualize_markers.py" output="screen"/>
  
</launch>

